manifests/instance.pp

```diff
@@ -112,6 +118,14 @@
     $instance_configdir = $configdir
   }
 
+  # manage_dir setting
```

Comment:
Alright, unless I'm going slightly 1.30am crazy.

The user may choose to set $manage_dir to false using:

```
elasticsearch::instance { 'es-01':
  manage_datadir => false,
}
```

In that case, $elasticsearch::manage_datadir will be true and $manage_datadir will be false, and we'll fall through to the 'else'.

Still not seeing how I can remove this, as much as I wish I'd not had to put it in in the first place.